style.css

```diff
@@ -0,0 +1,378 @@
+body{
+  padding-right: 10px;
+  }
+  .youtube-page{
+    margin-top: 60px;
+    display: flex;
+    /* justify-content: space-between; */
+    font-family: 'Roboto', sans-serif;
+    height: 5000px;
+  }
+  
+  
```

Comment:
Kindly install linters in your setup and correct the errors

Reply:
- [ ] Great job on this review but you should give also some document on how to set up linters in the student project and give the reason why to use them :smile: